### DISCUSSION TITLE
Modes: Smart to Manual, can't go back to Smart

### DISCUSSION THREAD

**Discussion by bzx** [maca88/SmartBikeLights#13] [Q&A]
Hello,

On 530, I can't seem to go back to Smart mode from Manual mode, as described in the project page:
`When in Smart mode the following can be done to switch control modes:
To go into Manual control mode, use the physical button on the light to switch the light mode. For restoring back to the Smart mode, switch to a different data screen and then back to the original one.`

Repro steps:
- lights are in Smart mode (S is shown in the data field for both)
- I start the activity, both lights turn on according to my config, data field shows their mode
- I press the button on the front light to change its mode, now the screen shows "M" and the mode for the front light
- I switch to a different data screen (for example I have a map screen), and back to the screen with data fields for the lights
- Front light mode is still set to "M".

My config:
`#4587520,196641#2,2|:1:0:1H[2|:1:63:1D=1#6291461,1409482753#2,2|:1:0:1H[2|:1:7:1D=1#3:Ion 200 RT|Off:0|Day Flash:63|Night Flash:62#4:Flare RT|Off:0|Day Flash:7|Night Flash:63|Night Steady:5#0::#0:0#B3121#2#1#0#0`

Is this a known issue?

EDIT:
Ah crap, this is in the section for "Other devices"... ok...
Any plans to add this to 530 and the like? ;)

**Comment by maca88** [comment]
Hi,

for Edge 530 you need to follow the steps `For Edge 530` described in section [Changing control modes](https://github.com/maca88/SmartBikeLights#changing-control-modes). As Edge 530 has CIQ 3.2, you can change the control mode with the menu and the workaround by switching between data screens is only available for older devices.
